infrastructure/src/main/java/org/corfudb/infrastructure/FailureHandlerDispatcher.java

```diff
@@ -205,7 +238,10 @@ private void reconfigureSequencerServers(CorfuRuntime runtime, Layout originalLa
                 verifyStreamTailsMap(streamTails);
 
                 // Configuring the new sequencer.
-                newLayout.getSequencer(0).reset(maxTokenRequested + 1, streamTails).get();
```

Comment:
this is precisely the last comment I made, but then we can remove the tail queries altogehter.

note this means that fastSmrMapsLoader becomes crucial for reconfiguration with a new sequencer.